{
  "debianbug": 737778,
  "description": "__init__.py in f2py in NumPy before 1.8.1 allows local users to write to arbitrary files via a symlink attack on a temporary file.",
  "releases": {
    "buster": {
      "fixed_version": "1:1.8.1~rc1-1",
      "repositories": {
        "buster": "1:1.16.2-1"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "stretch": {
      "fixed_version": "1:1.8.1~rc1-1",
      "repositories": {
        "stretch": "1:1.12.1-3"
      },
      "status": "resolved",
      "urgency": "low"
    }
  },
  "scope": "local"
}